esp/mlrs-wifi-bridge/mlrs-wifi-bridge.ino

```diff
@@ -80,6 +84,7 @@ int wifi_channel = 13;
 // comment all for default behavior, which is using only Serial port
 //#define USE_SERIAL_DBG1 // use Serial for communication and flashing, and Serial1 for debug output
 //#define USE_SERIAL1_DBG // use Serial1 for communication, and Serial for debug output and flashing
+//#define USE_SERIAL2_DBG // use Serial2 for communication, and Serial for debug output and flashing
```

Comment:
adding this option should mean that in the boards.h file there should be also plenty of #undef USE_SERIAL2_DBG in all the other modules, shouldn't it

Reply:
Done.